LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDevice.cs

```diff
@@ -421,6 +431,78 @@ internal async Task<bool> UpdateAfterJoinAsync(string devAddr, string nwkSKey, s
             return succeeded;
         }
 
+        internal void SetRequestHandler(ILoRaDataRequestHandler dataRequestHandler) => this.dataRequestHandler = dataRequestHandler;
```

Comment:
property?

Reply:
There is no getter. Made the method to enable easier unit testing

Reply:
True